recipes/poco/all/conanfile.py

```diff
@@ -48,7 +48,7 @@ class PocoConan(ConanFile):
         "PocoMongoDB": _PocoComponent("enable_mongodb", True, ("PocoNet", ), True),
         "PocoNet": _PocoComponent("enable_net", True, ("PocoFoundation", ), True),
         "PocoNetSSL": _PocoComponent("enable_netssl", True, ("PocoCrypto", "PocoUtil", "PocoNet", ), True),    # also external openssl
-        "PocoNetSSLWin": _PocoComponent("enable_netssl_win", True, ("PocoNet", "PocoUtil", ), True),
+        "PocoNetSSLWin": _PocoComponent("enable_netssl_win", False, ("PocoNet", "PocoUtil", ), True),
```

Comment:
for reviewers: it's not obvious from the diff itself what does True/False mean here
it's just a default option value according to the: 
https://github.com/conan-io/conan-center-index/blob/1ba9584da61a7119ac3f26d91ec0a6e7a9380dba/recipes/poco/all/conanfile.py#L32